### PR TITLE
Fixed PropType error on IconButton

### LINKED ISSUE #17789: PropType error on IconButton
### Description
Getting a proptype error on `develop` for IconButton could be apart of the recent icon updates

Screenshot 2023-02-16 at 3 54 48 PM

### PR DESCRIPTION
**Pull request by NidhiKJha**
## Explanation
* Fixes #17789 

### After
ss

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/helpers/utils/permission.js [hunk 1]
```diff
@@ -49,7 +52,13 @@ const PERMISSION_DESCRIPTIONS = deepFreeze({
...
+        leftIcon: (
+          <Icon
+            name={ICON_NAMES.EYE}
+            margin={4}
+            color={Color.iconAlternative}
+          />
+        ),
         rightIcon: null,
         weight: 1,
       };
```
Comment (georgewrmarshall) [inline ui/helpers/utils/permission.js:64]: Would it be possible to add a screenshot here for a visual check?

File: ui/helpers/utils/permission.js [hunk 2]
```diff
@@ -49,7 +52,13 @@ const PERMISSION_DESCRIPTIONS = deepFreeze({
...
     permissionValue.caveats[0].value.map(({ path, curve }) => {
       const baseDescription = {
-        leftIcon: 'fas fa-eye',
+        leftIcon: (
+          <Icon
+            name={ICON_NAMES.EYE}
+            margin={4}
+            color={Color.iconAlternative}
+          />
+        ),
```
Comment (georgewrmarshall) [inline ui/helpers/utils/permission.js:61]: Would it be possible to add a screenshot here for a visual check?

**Review by georgewrmarshall — COMMENTED** [review]
Wondering if the solution here is to change the propType or to update the component that is being used? If it works visually(would be good to get some screenshots) then I think we could update the proptypes

File: ui/helpers/utils/permission.js (hunk 2 above)
Comment (NidhiKJha) [reply ui/helpers/utils/permission.js]: Ah, my bad. This file should not be here. I updated it on wrong branch. It's actually for this [PR](https://github.com/MetaMask/metamask-extension/pull/17800). Screenshot is there as well 😄

**Review by georgewrmarshall — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**